qa-pipelines/tasks/cf-deploy-upgrade-common.sh

```diff
@@ -60,23 +60,46 @@ is_namespace_ready() {
         | sort \
         | uniq) ]]
 }
+wait_for_jobs() {
+    local release=$1
+    local jobs_desired_remaining
+    start=$(date +%s)
+    for (( i = 0 ; i < 480 ; i ++ )); do
+        # Get the list of all jobs in the helm release, and subtract the value of 'completed' from 'desired'
+        # It would be better to parse this from `helm get manifest`, but since that command is broken in helm 
+        # v2.8.2 (https://github.com/helm/helm/issues/3833) for now we'll parse it from the human-readable status
+        jobs_desired_remaining=$(helm status $release | awk '/==> v1\/Job/ { getline; getline; while (NF>0) { print $2 - $3; getline } }')
+        now=$(date +%s)
+        if [[ $(echo "${jobs_desired_remaining}" | sort -n | tail -1) -le 0 ]]; then
```

Comment:
Shouldn't that be `-gt`? You have (desired - successful), so it should never be negative.

Reply:
Oh, because I'm an idiot and read the condition backwards; `-le 0` for accept.